code/commands/view-commands.cpp

```diff
@@ -504,7 +504,11 @@ void Gobby::ViewCommands::on_menu_language_changed()
 
 		// The language should exist by construction, if the languages
 		// available in the language manager don't change at runtime
-		g_assert(language != NULL);
+		if(language != NULL) {
```

Comment:
This should be `language == NULL` or you are doing the exact opposite of your intention :)